multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -331,7 +331,7 @@ fn assert_balances(
     }
 }
 
-fn deploy_smart_contract(wallet: &Wallet, web3: &Web3<Http>, chain_id: u8) -> Address {
+fn deploy_smart_contract(wallet: &Wallet, web3: &Web3<Http>, chain: Chain) -> Address {
```

Comment:
It occurs to me that this may still be the SUB smart contract. We may want to consider updating this to the actual MASQ smart contract in case someone checks. Not to be done in this card but I may create a new card after some discussion.

Reply:
Please, do it, I'd appreciate if it were you who made the card.

Reply:
I created an issue using the "Reference in new issue" function. I'll try to convert to a card next.